code/modules/lighting/lighting_atom.dm

```diff
@@ -48,9 +48,6 @@
 		else
 			light = new/datum/light_source(src, .)
 
-/atom/proc/extinguish_light()
-	return
```

Comment:
this is used by vampires in the vampire rework. just as a note.

Reply:
:|

Reply:
I am sorry :C